frontend/public/components/_dropdown.scss

```diff
@@ -238,6 +238,8 @@ $dropdown-background--hover: $color-pf-black-200; // pf-c-dropdown__menu-item--h
   border-width: 0 !important; // TEMP fix until https://github.com/patternfly/patternfly-next/issues/2019 is fixed upstream
```

Comment:
Unrelated to this fix, but can we remove this rule now @rhamilto @sg00dwin @jcaianirh ? (We should do that as a separate PR.)

Reply:
Yes, this override can be removed now.  Anyone want to tackle that?